indico_hub/schemas.py

```diff
@@ -3,3 +3,46 @@
 
 
 # TODO: Implement Marshmallow schemas for the API request data
+
+from flask_marshmallow import *
+from marshmallow import *
+from marshmallow_sqlalchemy import *
+from webargs.core import T
+from .models import *
+"""
+payload = {
+        'url': BASE_URL,
+        'contact': contact,
+        'email': email,
+        'organization': "it"
+    }
+"""
+class instanceSchema(SQLAlchemyAutoSchema):
```

Comment:
PEP-8 suggests UpperCamelCase for class names (ie `InstanceSchema`)

Reply:
solved